pom.xml

```diff
@@ -464,7 +464,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-deploy-plugin</artifactId>
-                    <version>2.8.2</version>
+                    <version>3.0.0-M1</version>
```

Comment:
Should we really be using milestone (i. e. pre-release) versions of Maven plugins?

Reply:
Probably not, I was just testing what worked and what didn't.  I need to fix the new Nullaway errors in this PR still.